### PR TITLE
2.0

### PR DESCRIPTION
**Pull request by LevaniVashadze**

### CODE REVIEW

**Review by OpenSourceSimon — APPROVED** [review]
Haven't tested but looks good to me!

**Comment by LevaniVashadze** [comment]
@JasonLovesDoggo these queries dont work commented it out
image

**Comment by LevaniVashadze** [comment]
the IF NOT is not working

**Comment by JasonLovesDoggo** [comment]
> @JasonLovesDoggo these queries dont work commented it out
> image
> 

What makes you think that they don't work..

**Comment by JasonLovesDoggo** [comment]
Works perfectly for me

**Comment by LevaniVashadze** [comment]
> > @JasonLovesDoggo these queries dont work commented it out
> > image
> > 
> 
> What makes you think that they don't work.. 

It crashes... And i left a comment with the error

**Comment by LevaniVashadze** [comment]
Btw please dont alter the files in the panel, today it couldnt pull at all, Freebie needed to jump trough some hoops to get it cleaned up. I did copy over the stuff into the version control version.

**Comment by JasonLovesDoggo** [comment]
> Btw please dont alter the files in the panel, today it couldnt pull at all, Freebie needed to jump trough some hoops to get it cleaned up. I did copy over the stuff into the version control version. 

It's super easy to fix such things Just copy paste remote files or just force pull

**Comment by LevaniVashadze** [comment]
1. Shouldn't need to do that and changes might get lost.

2. Running commands doesn't work me and freebie tried, need to modify the startup cmd, wich i cant do.

Thank you for your understanding

File: cogs/Birthdays.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+from utils.shortcuts import QuickEmb, sucEmb, errorEmb
+from utils.CONSTANTS import months, congrats_messages
+from utils.bot import OGIROID
+
+
+class Birthday(commands.Cog):
+    def __init__(self, bot: OGIROID):
+        self.bot = bot
+        self.birthday: BirthdayHandler = None
+        self.birthday_check.start()
```
Comment (JasonLovesDoggo) [inline cogs/Birthdays.py:19]: Move that into on_ready

**Review by JasonLovesDoggo — APPROVED** [review]

**Pull request merged**